### PR TITLE
[MODFEE-13] Add fee/fine tokens to patron notices

### PR DESCRIPTION
**Pull request by dmytrotur**
## Purpose
Add fee/fine tokens to patron notices
Resolves: [MODFEE-13](https://issues.folio.org/browse/MODFEE-13)

### CODE REVIEW

File: src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+public class FeeFineNoticeContext {
+
+  private static final String PAID_FULLY = "Paid fully";
+  private static final String PAID_PARTIALLY = "Paid partially";
+
+  private Owner owner;
+  private Feefine feefine;
+  private Account account;
+  private Feefineaction feefineaction;
```
Comment (kostyantyn-kh) [inline src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java]: Could this be renamed to FeeFineAction?

File: src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    this.feefine = feefine;
+    this.account = account;
+    this.feefineaction = feefineaction;
+  }
+
+  public String getTemplateId() {
+    String templateId = null;
+
+    if (feefineaction.getPaymentMethod() == null) {
+      templateId = feefine.getChargeNoticeId() != null ?
```
Comment (kostyantyn-kh) [inline src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java]: Could be simplified: return feefine...

File: src/main/java/org/folio/rest/repository/OwnerRepository.java [hunk 3]
```diff
@@ -15,9 +16,9 @@ public OwnerRepository(PostgresClient pgClient) {
     this.pgClient = pgClient;
   }
 
-  public Future<Owner> getById(String id) {
+  public Future<FeeFineNoticeContext> loadOwner(FeeFineNoticeContext context) {
     Future<Owner> future = Future.future();
-    pgClient.getById(OWNERS_TABLE, id, Owner.class, future);
-    return future;
+    pgClient.getById(OWNERS_TABLE, context.getFeefine().getOwnerId(), Owner.class, future);
```
Comment (kostyantyn-kh) [inline src/main/java/org/folio/rest/repository/OwnerRepository.java]: Probably extra check for null is needed here (same for other repository files)

File: src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java (hunk 2 above)
Comment (dmytrotur) [reply src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java]: Done

File: src/main/java/org/folio/rest/repository/OwnerRepository.java (hunk 3 above)
Comment (dmytrotur) [reply src/main/java/org/folio/rest/repository/OwnerRepository.java]: Done

**Pull request merged**